src/wot.py

```diff
@@ -0,0 +1,34 @@
+import os
+from tabulate import tabulate
+from collections import OrderedDict
+
+from network_tools import *
+from tools import *
+from constants import *
+
+
+def received_sent_certifications(ep, id):
+    """
+    check id exist
+    many identities could exist
+    retrieve the one searched
+    get id of received and sent certifications
+    display on a chart the result with the numbers
+    """
+    if get_pubkeys_from_id(ep, id) == NO_MATCHING_ID:
+        print(NO_MATCHING_ID)
+        sys.exit(1)
+    certs = request(ep, "wot/lookup/" + id)["results"]
+    for cert in certs:
+        if cert["uids"][0]["uid"].lower() == id.lower():
+           certs = cert
```

Comment:
I believe a break is needed here to come out of the loop.
What if the id is not found?
I believe we could also be a little permissive and if several ids match the queried ID we could give information for all of them (but that could be a different enhancement later).